doc/src/bond_cs.md

```diff
@@ -315,18 +315,21 @@ See also the following example:
 Input and output streams
 ========================
 
-The input and output for binary protocols is provided by `IInputStream` and
-`IOutputStream` interfaces. Bond comes with standard implementations of these
-interfaces for memory buffers and `System.IO.Stream`, and applications can
-provide their own custom implementations.
-
-The `OutputBuffer` class implements `IOutputStream` interface on top of
+The input and output for binary protocols is provided by the
+[`IInputStream`](https://github.com/Microsoft/bond/blob/master/cs/src/core/io/IInputStream.cs)
```

Comment:
Links are good, but if we're gonna go there, can we also link the first mentions of `{Input,Output}{Buffer,Stream}`?